packages/expo-updates/ios/EXUpdates/EXUpdatesAppController.m

```diff
@@ -251,36 +250,68 @@ - (void)appLoaderTask:(EXUpdatesAppLoaderTask *)appLoaderTask didFinishBackgroun
   }
 }
 
-# pragma mark - internal
+# pragma mark - EXUpdatesAppController+Internal
 
-- (EXUpdatesConfig *)_loadConfigFromExpoPlist
+- (BOOL)initializeUpdatesDirectoryWithError:(NSError ** _Nullable)error
 {
-  NSString *configPath = [[NSBundle mainBundle] pathForResource:EXUpdatesConfigPlistName ofType:@"plist"];
-  if (!configPath) {
-    @throw [NSException exceptionWithName:NSInternalInconsistencyException
-                                   reason:@"Cannot load configuration from Expo.plist. Please ensure you've followed the setup and installation instructions for expo-updates to create Expo.plist and add it to your Xcode project."
-                                 userInfo:@{}];
+  NSError *err;
+  _updatesDirectory = [EXUpdatesUtils initializeUpdatesDirectoryWithError:&err];
+  if (err && error) {
+    *error = err;
   }
```

Comment:
I'd just thread through the error pointer and rely on the convention that there is no error iff the return value is non-nil/non-false unless clearly specified.
 
```suggestion
  _updatesDirectory = [EXUpdatesUtils initializeUpdatesDirectoryWithError:error];
  return _updatesDirectory != nil;
```